### PR TITLE
Adds mute quirk

### PR DESCRIPTION
**Pull request by Hardly3D**
## About The Pull Request
This PR adds a new negative quirk which makes you mute, however you are still able to emote and use your PDA to message people.

## Why It's Good For The Game

We got paraplegic, blind, and deaf available as a quirk, so mute would be a fine addition. And it would also be good for roleplay as well. It could also encourage some players to try roleplaying as a mute character.

## Changelog
:cl: Hardly
add: Added new quirk: Mute
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/processing/quirks.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	if(!quirks.len)
 		SetupQuirks()
 
-	quirk_blacklist = list(list("Blind","Nearsighted"),list("Jolly","Depression","Apathetic","Hypersensitive"),list("Ageusia","Vegetarian","Deviant Tastes"),list("Ananas Affinity","Ananas Aversion"),list("Alcohol Tolerance","Light Drinker"),list(list("Neat","NEET")))
+	quirk_blacklist = list(list("Blind","Nearsighted"),list("Jolly","Depression","Apathetic","Hypersensitive"),list("Ageusia","Vegetarian","Deviant Tastes"),list("Ananas Affinity","Ananas Aversion"),list("Alcohol Tolerance","Light Drinker"),list("Social Anxiety","Mute"),list("Neat","NEET"))
```
Comment (Kapu1178) [inline code/controllers/subsystem/processing/quirks.dm]: while we're here...
```suggestion
	quirk_blacklist = list(
		list("Blind","Nearsighted"),
		list("Jolly","Depression","Apathetic","Hypersensitive"),
		list("Ageusia","Vegetarian","Deviant Tastes"),
		list("Ananas Affinity","Ananas Aversion"),
		list("Alcohol Tolerance","Light Drinker"),
		list("Social Anxiety","Mute"),
		list("Neat","NEET")
	)
```

**Comment by Crossedfall** [comment]
I mean.... They could just play mime. 🤷‍♂️ 

Speaking of, wouldn't this basically just be two free points for mime players? I don't really think this needs to be a quirk. Unlike blindness or really any of the other handicaps, it's pretty easy to roleplay as a mute character without this actually being implemented.

**Comment by super12pl** [comment]
What crossed said mostly
We don't need hard mechanics for every roleplay gimmick nor should we give people more free quirk points.

**Comment by BrumalLake** [comment]
pretty sure that breaking your vow of silence would let you talk anyways

**Comment by Hardly3D** [comment]
> 
> 
> I mean.... They could just play mime. 🤷‍♂️
> 
> Speaking of, wouldn't this basically just be two free points for mime players? I don't really think this needs to be a quirk. Unlike blindness or really any of the other handicaps, it's pretty easy to roleplay as a mute character without this actually being implemented.

Though mime is limited to one slot and they can only use emoticons for their PDA, not to mention that they have a job to entertain the crew.

The concern of mimes using this to powergame quirks is also a good point, but it may be worth noting that mime players already sorta powergame quirks by taking social anxiety for the free point and there really isn't much to stop them from doing that. We could try decreasing the amount of quirk points mute gives from 2 to 1, it's already incompatible with social anxiety either way.

IMO, while we can just RP as a mute right now, the quirk could still be useful for the roleplay (eg. shows that you're a mute when examining, preventing accidental talking, etc.)

**Comment by Hardly3D** [comment]
> 
> 
> pretty sure that breaking your vow of silence would let you talk anyways

Just checked, you are unable to talk even if you take mutadone or try to break your vow
dreamseeker_easLwkFNZs

**Comment by utilbird** [comment]
> The concern of mimes using this to powergame quirks is also a good point, but it may be worth noting that mime players already sorta powergame quirks by taking social anxiety for the free point and there really isn't much to stop them from doing that. We could try decreasing the amount of quirk points mute gives from 2 to 1, it's already incompatible with social anxiety either way.

One oversight doesn't justify adding another

**Comment by williamcll** [comment]
pretty sure mimes can talk through a ventriloquist doll

**Comment by ike709** [comment]
There's too many ways to bypass this for a mediocre gimmick.

**Pull request closed without merging**